src/backends/ilp-quoter/index.js

```diff
@@ -24,6 +24,7 @@ class ILPQuoter {
     this.currencyPairs = this.currencyWithLedgerPairs.map((p) => [p[0].slice(0, 3),
                                                                   p[1].slice(0, 3)])
     this.backendUri = opts.backendUri
+    this.backendStatus = 'KO'
```

Comment:
I remember seeing "KO" somewhere recently, but I agree it is confusing. How about "NOT READY"?